packages/labs/analyzer/src/lib/javascript/jsdoc.ts

```diff
@@ -47,7 +47,7 @@ const parseNameTypeDescRE =
 
 // Regex for parsing type, name, and description from JSDoc comments
 const parseTypeNameDescRE =
-  /^\{(?<type>[^}]+)\}\s+\[?(?<name>[^{}[\]\s=]+)(?:=(?<defaultValue>[^\]]+))?\]?(?:\s+-\s+)?(?<description>[\s\S]*)$/m;
+  /^\{(?<type>.+)\}\s+\[?(?<name>[^{}[\]\s=]+)(?:=(?<defaultValue>[^\]]+))?\]?(?:\s+-\s+)?(?<description>[\s\S]*)$/m;
```

Comment:
I think we're in luck here in that CSS property type syntax doesn't include `}`, but _event_ type syntax definitely can. Do we have tests for that? I think we'll need a way to escape `}`.

No need to fix or change anything now, but let's come back to that.

Reply:
Well they're all on the same code path now, so I think the existing tests should cover it, no?

Reply:
Yeah, we have that [here](https://github.com/lit/lit/blob/b9c3443185ef2f7bbe728dc61eb4b13b7a0dccb4/packages/labs/analyzer/test-files/js/events/element-a.js#L34) so we're good!